main.tf

```diff
@@ -0,0 +1,450 @@
+locals {
+  common_tags = {
+    itse_app_env      = var.itse_app_env
```

Comment:
The dilemma of public repos is whether or not to include code specific to our team (e.g. `itse_app_env`). One way to handle this is to add a variable that accepts our preferred tags. But that's more work for us when we deploy this code. Everything has tradeoffs. But in reality, this repo is very unlikely to be of use to anyone but us. So I guess it's ok to leave this as you have it. Sorry for rambling!